### PR TITLE
feat(whois): add options

### PR DESCRIPTION
**Pull request by SeparateRecords**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Removes non-existent options and adds all the other options

**What is the current behavior? (You can also link to an open issue here)**
Incorrect options are suggested, and none of the actual options are

**Additional info:**
Because macOS, it's another BSD spec. Will need to be updated in the future.

### CODE REVIEW

File: src/whois.ts [hunk 1]
```diff
@@ -1,13 +1,95 @@
 const completionSpec: Fig.Spec = {
   name: "whois",
+  description: "Query a database for information about a domain registrant",
   args: {
     name: "name",
+    isVariadic: true,
   },
-  description: "Client for the whois directory service",
   options: [
-    { name: ["-H", "-p", "PORT"], description: "Connect to PORT" },
```
Comment (QuiiBz) [inline src/whois.ts:8]: Looks like the `-p` option exists, along with `-h` for host:
Screenshot 2021-09-22 at 12 41 19

Could you add them back?

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: src/whois.ts (hunk 1 above)
Comment (SeparateRecords) [reply src/whois.ts:8]: Already added (L39 and L69) 😄 

Although, I didn't notice that was a pipe between `-c` and `-h` - they are indeed exclusive, so I'll fix that.

Comment (QuiiBz) [reply src/whois.ts:8]: I'm dumb, sorry 😅 Alright!

**Comment by SeparateRecords** [comment]
Btw this is ready for review again 😅 I probably should have said so sooner haha

**Review by fedeci — APPROVED** [review]

**Review by QuiiBz — APPROVED** [review]
LGTM, a big improvement on this spec, thank you so much!

**Pull request merged**